tree/dataframe/inc/ROOT/RDF/RVariedAction.hxx

```diff
@@ -148,10 +153,10 @@ public:
       SetHasRun();
    }
 
-   /// Return the partially-updated value connected to the nominal result.
+   /// Return the partially-updated value connected to the first upstream varied filter.
    void *PartialUpdate(unsigned int slot) final { return PartialUpdateImpl(slot); }
 
-   /// Return the per-sample callback connected to the nominal result.
+   /// Return the per-sample callback connected to the first upstream varied filter.
    ROOT::RDF::SampleCallback_t GetSampleCallback() final { return fHelpers[0].GetSampleCallback(); }
```

Comment:
uhm, not for this PR, but this does not make much sense :sweat_smile: 

Either all the helpers for this action need to register a callback, or none. I'll propose a patch to at least make sure we don't silently register just _some_ of he callbacks.